appleseed-Installer/appleseed-installer.wixproj

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" InitialTargets="EnsureWixToolsetInstalled" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProductVersion>3.10</ProductVersion>
+    <ProjectGuid>685fba7f-4a79-497b-86f6-ac23f94bca89</ProjectGuid>
+    <SchemaVersion>2.0</SchemaVersion>
+    <OutputName>AppleSeed</OutputName>
+    <OutputType>Package</OutputType>
+    <Name>appleseed-installer</Name>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <IntermediateOutputPath>obj\$(Configuration)\</IntermediateOutputPath>
+    <DefineConstants>Debug</DefineConstants>
+    <SuppressIces>ICE20</SuppressIces>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <IntermediateOutputPath>obj\$(Configuration)\</IntermediateOutputPath>
+    <SuppressIces>ICE20</SuppressIces>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="ErrorDialog.wxs" />
+    <Compile Include="FinishedDialog.wxs" />
+    <Compile Include="LicenseAgreeDlg.wxs" />
+    <Compile Include="PathSelectDialog.wxs" />
+    <Compile Include="SetupDialog.wxs" />
+    <Compile Include="Product.wxs" />
+    <Compile Include="ProgressDialog.wxs" />
+    <Compile Include="VersionSelectDlg.wxs" />
+  </ItemGroup>
+  <ItemGroup>
+    <WixExtension Include="WixUIExtension">
+      <HintPath>$(WixExtDir)\WixUIExtension.dll</HintPath>
+      <Name>WixUIExtension</Name>
+    </WixExtension>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Resources" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Resources\Bg.png" />
```

Comment:
What is the license for these images?

Reply:
I choose the left image from appleseed website gallery, but there was not an author or trademark to explain anything.
The other parts are created by me and completely free with no limitation.